### ISSUE TITLE
Could you provide the backbone pretrain-weights based on imagenet ?

### ISSUE THREAD

**Issue by FDInSky** [ultralytics/yolov5#1571]
## ❔Question

When I want to train yolov5 on coco dataset, I usually use the imagenet pretrain for backbone.
Does YOLOv5 directly use yolov5.pt? Is it right?

**Comment by zhiqwang** [comment]
I’m also confused about this issue

**Comment by glenn-jocher** [comment]
@FDInSky @zhiqwang commands to reproduce our results training COCO are displayed in README. No pretrained weights are used.
https://github.com/ultralytics/yolov5#training

**Comment by zhiqwang** [comment]
> No pretrained weights are used.

Hi @glenn-jocher ,
This result is impressive and superb, maybe just this could be concluded to one thesis 😄

**Comment by glenn-jocher** [comment]
@zhiqwang it's possible to start from pretrained weights also, though I haven't experimented as much with this on COCO (for smaller datasets I highly recommend starting from pretrained weights, for larger datasets it may not matter much). For example you could train COCO starting from the already-trained COCO weights. I'm not sure what the result would be, but I suspect you'd want a slightly different setting for best results there, like perhaps 100 epochs of training rather than 300, as this will lead to faster overfitting. The command might be:

`python train.py --data coco.yaml --weights yolov5s.pt --epochs 100`

**Comment by zhiqwang** [comment]
Hi @glenn-jocher , Thank you. I had just trained my first COCO detection models with your default configs (`yolov5s`, get map 0.37). I'll do more experiments as you suggest!!

**Comment by glenn-jocher** [comment]
@zhiqwang ah, that's good you are able to reproduce our results. I believe the default values will generally produce the best results (that's why we have them as defaults), but you can also simply view them as a good starting point for your own research. 

There's a lot of ways to modify the results, in areas such as augmentation, loss function, loss balancing, hyperparameters and of course architecture.

**Comment by zhiqwang** [comment]
Hi @glenn-jocher , Thanks again!!

**Comment by ghost** [comment]
Hi @glenn-jocher, Why not try the imagenet pre-training for the backbone? As YOLO always make prediction of the bounding box upon the category of the related class, such the pretrained model is more likely to produce a related faster convergence, especially in the early stage of the training, right? To me, it is very likely to improve the entire model's performance...

**Comment by glenn-jocher** [comment]
@Sun-Rider YOLOv5 COCO pretrained weights are provided and downloaded automatically:
`python train.py --weights yolov5s.pt`

**Comment by zhiqwang** [comment]
Hi @Sun-Rider and @glenn-jocher , Actually in [ultralytics/yolov3](https://github.com/ultralytics/yolov3), the model are trained with imagenet pretrained backbonne, so here we can think of yolov5 as an upgraded version?

**Comment by glenn-jocher** [comment]
@zhiqwang no models in ultralytics/yolov3 or ultralytics/yolov5 use pretrained weights. All official YOLOv3 and YOLOv5 models produced by Ultralytics are trained from scratch, i.e. `--weights ''`. Directions to reproduce training are displayed in respective READMEs.
